dspace-spring-rest/src/main/java/org/dspace/app/rest/BitstreamContentRestController.java

```diff
@@ -125,6 +125,40 @@ public void retrieve(@PathVariable UUID uuid, HttpServletResponse response,
 		}
 	}
 
+	private Pair<InputStream, Long> getBitstreamInputStreamAndSize(Context context, Bitstream bit) throws SQLException, IOException, AuthorizeException {
+
+		if (citationDocumentService.isCitationEnabledForBitstream(bit, context)) {
+			return generateBitstreamWithCitation(context, bit);
```

Comment:
This PR was discussed in today's DSpace 7 meeting. As discussed there (with @abollini and others), we need to have a way to still download the *original* bitstream, even when the Cover Page feature is enabled.  For example, an Administrative user needs to be able to see the original bitstream from the Admin UI, but they may want to also optionally see how the file will look with the Cover Page applied.

We haven't yet decided on the best way to implement this...if the cover page should be a separate endpoint, or maybe there's some sort of parameter that Administrative users can use to still get the original bitstream?  Other thoughts welcome.

Meeting notes at: https://wiki.duraspace.org/display/DSPACE/2018-02-01+DSpace+7+Working+Group+Meeting+notes

Reply:
This was ported from DSpace 6: https://github.com/DSpace/DSpace/blob/dspace-6.2/dspace-xmlui/src/main/java/org/dspace/app/xmlui/cocoon/BitstreamReader.java#L369

The `CitationDocumentService` will not add a cover page for admins: https://github.com/DSpace/DSpace/blob/dspace-6.2/dspace-api/src/main/java/org/dspace/disseminate/CitationDocumentServiceImpl.java#L240

So while I agree with the above use case, I think it is an optional nice to have. This PR complies with the DSpace 6 behaviour.

Reply:
I see...looks like it was a misunderstanding around what the `isCitationEnabledForBitstream()` method does.  It never returns `true` for Admin users.